annotationsx/settings/aws.py

```diff
@@ -168,6 +173,10 @@
 
 # should be changed depending on platform roles, these are set up for edX
 ADMIN_ROLES = ['urn:lti:instrole:ims/lis/Administrator', 'urn:lti:role:ims/lis/Instructor', 'urn:lti:role:ims/lis/TeachingAssistant']
+
+#TODO: check if this is what we want
+ADMIN_ROLES += ['Administrator', 'Instructor', 'TeachingAssistant']
+
```

Comment:
I'm not completely certain what the difference is between ext_roles vs roles, although it seems to include the full URN for each role, whereas roles doesn't always include the full URN.

Having said that, the "roles" value can include more than one value. For me, since I'm an administrator, I have this:

```roles: "Instructor,urn:lti:instrole:ims/lis/Administrator"```

So make sure that any logic that tests roles can handle more than one role value. That is, you shouldn't do this:

```roles == "Instructor"```

Since that will fail for me.